chunk/chunk_store.go

```diff
@@ -47,6 +47,17 @@ var (
 		Help:      "Time spent doing S3 requests.",
 		Buckets:   []float64{.025, .05, .1, .25, .5, 1, 2},
 	}, []string{"operation", "status_code"})
+	rowWrites = util.NewHashBucketHistogram(util.HashBucketHistogramOpts{
+		HistogramOpts: prometheus.HistogramOpts{
+			Namespace: "cortex",
+			Name:      "chunk_store_row_write_total",
```

Comment:
`_total` is a suffix reserved for counters by convention, not for histograms, which get additional `_sum` and `_count` counters created automatically. I'd call this `chunk_store_row_writes_distribution` or something.